From 382253d87dab98a7d082cd91e40eb59c8b70077b Mon Sep 17 00:00:00 2001
From: Noa Osherovich <noaos@mellanox.com>
Date: Mon, 28 Oct 2019 15:58:12 +0200
Subject: [PATCH rdma-core 03/13] man: Fix wrong field in ibv_wr_post's man
 page

[ Upstream commit 5850789bb89df8f418d99b5dfb7f18bc3dd6cf58 ]

The example in the man page refers to a non-existing field.
Update to the correct field name.

Fixes: 58ef962809865 ('verbs: Introduce a new post send API')
Signed-off-by: Noa Osherovich <noaos@mellanox.com>
Signed-off-by: Nicolas Morey-Chaisemartin <nmoreychaisemartin@suse.com>
---
 libibverbs/man/ibv_wr_post.3.md | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libibverbs/man/ibv_wr_post.3.md b/libibverbs/man/ibv_wr_post.3.md
index 4d5f80d6..ab7fc5f5 100644
--- a/libibverbs/man/ibv_wr_post.3.md
+++ b/libibverbs/man/ibv_wr_post.3.md
@@ -315,7 +315,7 @@ ibv_wr_set_sge(qpx, lkey, local_addr_1, length_1);
 
 /* create 2nd WRITE_WITH_IMM WR entry */
 qpx->wr_id = my_wr_id_2;
-qpx->send_flags = IBV_SEND_SIGNALED;
+qpx->wr_flags = IBV_SEND_SIGNALED;
 ibv_wr_rdma_write_imm(qpx, rkey, remote_addr_2, htonl(0x1234));
 ibv_set_wr_sge(qpx, lkey, local_addr_2, length_2);
 
-- 
2.20.1

